packages/scaffold-config/src/frameworks.ts

```diff
@@ -266,4 +273,23 @@ export const WIZARD_FRAMEWORKS = [
     componentIndexHtml: componentIndexHtmlGenerator(),
     specPattern: '**/*.cy.ts',
   },
+  {
+    type: 'svelte',
+    configFramework: 'svelte',
+    category: 'library',
+    name: 'Svelte.js',
+    detectors: [dependencies.WIZARD_DEPENDENCY_SVELTE],
+    supportedBundlers: [dependencies.WIZARD_DEPENDENCY_WEBPACK, dependencies.WIZARD_DEPENDENCY_VITE],
+    dependencies: (bundler: WizardBundler['type'], projectPath: string): Promise<DependencyToInstall[]> => {
+      return Promise.all([
+        getBundlerDependency(bundler, projectPath),
+        isDependencyInstalled(dependencies.WIZARD_DEPENDENCY_SVELTE, projectPath),
+      ])
+    },
+    codeGenFramework: 'svelte',
+    glob: '*.svelte',
+    mountModule: mountModule('cypress/svelte'),
+    supportStatus: 'alpha',
```

Comment:
Didn't we have a conversation around not using `alpha` for our newly-supported frameworks? Or was that just for Angular?

Reply:
I don't think we ever came to a good conclusion on why we would add `alpha`. My take it that we would do it as a safeguard to allow us to make breaking changes to the framework without forcing a breaking change of Cypress itself.

Reply:
I am fairly sure I remember this, too.

My recollection (and I share this opinion) is we just drop alpha and beta entirely - I think we can ship without a label.

We did this with Angular, too - I think Svelte should follow suit?

The only public API we've committed to across frameworks is `mount()` - and I'm sure this won't change.

Reply:
Based on our CT sync with Brian today, it seems like Alpha is the right move. Going to add this back